packages/components/README.md

```diff
@@ -254,6 +254,33 @@ const theme = createTheme(appTheme)
 </StyledEngineProvider>
 ```
 
+To apply custom colors that align with your branding while maintaining the SDS design configurations, use the `makeSdsSemanticAppTheme`. Provide it with a Colors object that follows the SDS model.
```

Comment:
Oh wow, really appreciate you adding the documentation!
This is an awesome PR! 😍🥳